spec/requests/api/v1/docs/articles_spec.rb

```diff
@@ -0,0 +1,111 @@
+require "rails_helper"
+require "swagger_helper"
+
+RSpec.describe "Api::V1::Docs::Articles", type: :request do
+  let(:organization) { create(:organization) } # not used by every spec but lower times overall
+  let(:tag) { create(:tag, :with_colors, name: "discuss") }
+  let(:article) { create(:article, featured: true, tags: "discuss") }
+  let(:api_secret) { create(:api_secret) }
+  let(:v1_headers) { { "Accept" => "application/vnd.forem.api-v1+json", "api-key" => api_secret.secret } }
```

Comment:
I fiddled a bit with these variables and realized (I believe) that `rswag` works with them to generate the files, maybe? Some magic is happening inside that gem... I'm both intrigued and happy we're using it 😄

Reply:
Cool! Yeah those couple of lines that you removed weren't needed, but the `tag`, `article` and `organization` do make a difference, right? Are those mock records used to generate the content of the example response "under the hood"?

Asking because I believe I removed them locally and the Open API JSON generated had empty responses... But it may have been a glitch on my end 🤔 

I mentioned "magic happening" in the good sense, which gave me the feeling `rswag` is powerful 😄